.gitignore

```diff
@@ -49,7 +49,6 @@ lib/
 
 # cmake
 CMakeCache.txt
-*.cmake
```

Comment:
you're not supposed to be generating cmake inside the same folder where cmake exist. you're supposed to generate out-of source. Problem is, your assets are not copied into the build location because you are assuming that CMAKE_BINARY_DIR == CMAKE_SOURCE_DIR. I will revert that